Index: libdbusmenu-qt-0.9.0/src/CMakeLists.txt
===================================================================
--- libdbusmenu-qt-0.9.0.orig/src/CMakeLists.txt	2011-08-30 11:40:07.000000000 +0200
+++ libdbusmenu-qt-0.9.0/src/CMakeLists.txt	2012-02-19 18:55:32.000000000 +0100
@@ -64,6 +64,7 @@
 set_target_properties(dbusmenu-qt PROPERTIES
     VERSION ${dbusmenu_qt_lib_VERSION}
     SOVERSION ${dbusmenu_qt_lib_SOVERSION}
+    INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib"
     )
 
 target_link_libraries(dbusmenu-qt
