From 702538eaa3315f3fcad9f1daea01e6a83928967b Mon Sep 17 00:00:00 2001
From: Clinton Stimpson <clinton@elemtech.com>
Date: Mon, 25 Jul 2011 12:30:30 -0600
Subject: [PATCH] Qt4: Fix reference of undefined variable when detecting frameworks on Mac OS X

---
 Modules/FindQt4.cmake |   26 +++++++++++++-------------
 1 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/Modules/FindQt4.cmake b/Modules/FindQt4.cmake
index 86fce9d..406807b 100644
--- a/Modules/FindQt4.cmake
+++ b/Modules/FindQt4.cmake
@@ -578,19 +578,6 @@ IF (QT_QMAKE_EXECUTABLE AND QTVERSION)
 
   ENDIF ()
 
-  IF (APPLE)
-    SET(CMAKE_FIND_FRAMEWORK_OLD ${CMAKE_FIND_FRAMEWORK})
-    IF (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
-      SET(QT_USE_FRAMEWORKS ON CACHE INTERNAL "" FORCE)
-      SET(CMAKE_FIND_FRAMEWORK FIRST)
-    ELSE (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
-      SET(QT_USE_FRAMEWORKS OFF CACHE INTERNAL "" FORCE)
-      SET(CMAKE_FIND_FRAMEWORK LAST)
-    ENDIF (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
-  ENDIF (APPLE)
-
-  _QT4_ADJUST_LIB_VARS(QtCore)
-
   # set QT_LIBRARY_DIR based on location of QtCore found.
   IF(QT_QTCORE_LIBRARY_RELEASE)
     GET_FILENAME_COMPONENT(QT_LIBRARY_DIR_TMP "${QT_QTCORE_LIBRARY_RELEASE}" PATH)
@@ -614,6 +601,17 @@ IF (QT_QMAKE_EXECUTABLE AND QTVERSION)
     SET(QT_BINARY_DIR ${qt_bins} CACHE INTERNAL "" FORCE)
   ENDIF (NOT QT_BINARY_DIR  OR  QT_QMAKE_CHANGED)
 
+  IF (APPLE)
+    SET(CMAKE_FIND_FRAMEWORK_OLD ${CMAKE_FIND_FRAMEWORK})
+    IF (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
+      SET(QT_USE_FRAMEWORKS ON CACHE INTERNAL "" FORCE)
+      SET(CMAKE_FIND_FRAMEWORK FIRST)
+    ELSE (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
+      SET(QT_USE_FRAMEWORKS OFF CACHE INTERNAL "" FORCE)
+      SET(CMAKE_FIND_FRAMEWORK LAST)
+    ENDIF (EXISTS ${QT_LIBRARY_DIR}/QtCore.framework)
+  ENDIF (APPLE)
+
   # ask qmake for the include dir
   IF (QT_LIBRARY_DIR AND (NOT QT_QTCORE_INCLUDE_DIR OR NOT QT_HEADERS_DIR OR  QT_QMAKE_CHANGED))
       _qt4_query_qmake(QT_INSTALL_HEADERS qt_headers)
@@ -904,6 +902,8 @@ IF (QT_QMAKE_EXECUTABLE AND QTVERSION)
 
   # Set QT_xyz_LIBRARY variable and add 
   # library include path to QT_INCLUDES
+  _QT4_ADJUST_LIB_VARS(QtCore)
+
   FOREACH(QT_MODULE ${QT_MODULES})
     _QT4_ADJUST_LIB_VARS(${QT_MODULE})
   ENDFOREACH(QT_MODULE)
-- 
1.7.0

