Index: cmake-2.8.5/Source/cmFindProgramCommand.cxx
===================================================================
--- cmake-2.8.5.orig/Source/cmFindProgramCommand.cxx	2011-07-08 14:21:46.000000000 +0200
+++ cmake-2.8.5/Source/cmFindProgramCommand.cxx	2012-02-25 17:50:11.000000000 +0100
@@ -175,6 +175,7 @@
     
     // And finally to a c++ string
     executable = bundlePath + "/Contents/MacOS/" + std::string(buffer);
+    CFRelease( executableURL );
     }
 
   // Any CF objects returned from functions with "create" or 
@@ -182,7 +183,6 @@
   CFRelease( bundlePathCFS );
   CFRelease( bundleURL );
   CFRelease( appBundle );
-  CFRelease( executableURL );
 #endif
 
   return executable;
