Index: attica-0.4.1/lib/CMakeLists.txt
===================================================================
--- attica-0.4.1.orig/lib/CMakeLists.txt
+++ attica-0.4.1/lib/CMakeLists.txt
@@ -120,6 +120,7 @@
         SOVERSION ${CMAKE_LIBATTICA_VERSION_MAJOR}.${CMAKE_LIBATTICA_VERSION_MINOR}
         LINK_INTERFACE_LIBRARIES ""
         DEFINE_SYMBOL ATTICA_LIB_MAKEDLL
+        INSTALL_NAME_DIR ${LIB_DESTINATION}
     )
 endif(ATTICA_STATIC_BUILD)
 target_link_libraries(attica ${QT_QTCORE_LIBRARY} ${QT_QTNETWORK_LIBRARY})
